src/index.js

```diff
@@ -0,0 +1,52 @@
+import './style.css';
+import '@fortawesome/fontawesome-free/js/fontawesome';
+import '@fortawesome/fontawesome-free/js/solid';
+import '@fortawesome/fontawesome-free/js/regular';
+import '@fortawesome/fontawesome-free/js/brands';
+
+const array = [
```

Comment:
- [ ] [OPTIONAL] I suggest you give different indexes even if we are not creating the items dynamically.